udata_gouvfr/templates/faq/third-party.html

```diff
@@ -0,0 +1,82 @@
+{% extends theme('faq/base.html') %}
+
+{% block faq_content %}
+<h1>Découvrir l’OpenData en tant qu’intégrateur</h1>
+
+<p>La plateforme <a href="https://www.data.gouv.fr/fr/">data.gouv.fr</a> vous offre plusieurs moyens plus ou moins avancés d’intégrer les données disponibles en open data sur vos propres sites :</p>
```

Comment:
On this page http://www.data.gouv.fr/fr/terms/ the first occurrence is data.gouv.fr and the second one is Data.gouv.fr.
It could be a good idea to make a choice and stick to it indeed.
I vote for data.gouv.fr which is the way the official logo is written